ament_cmake_nose/cmake/ament_add_nose_test.cmake

```diff
@@ -133,5 +133,3 @@ function(_ament_add_nose_test testname path)
     LABELS "nose"
   )
 endfunction()
-
-# TODO provide function to register all found tests separately
```

Comment:
Tests can already be labels within Python. No need for CTest to get involved for that.